src/main/java/com/minecolonies/coremod/event/ClientEventHandler.java

```diff
@@ -64,14 +63,13 @@ public void renderWorldLastEvent(@NotNull final RenderWorldLastEvent event)
 
         if (structure != null)
         {
-            final BlockPos position = Settings.instance.getPosition();
             if (Settings.instance.getStructureName().contains(AbstractEntityAIStructure.WAYPOINT_STRING))
             {
-                RenderUtils.renderWayPoints(position, world, event.getPartialTicks());
+                //TODO: Implement waypoint rendering.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)